ra/ra.go

```diff
@@ -440,6 +440,34 @@ func (ra *RegistrationAuthorityImpl) checkPendingAuthorizationLimit(ctx context.
 	return nil
 }
 
+// checkInvalidAuthorizationLimits checks the failed validation limit for each
+// of the provided hostnames, in parallel. It returns the first error.
```

Comment:
nit: I think this comment needs to be updated to reflect that the parallel lookup is only in effect when `features.ParalellCheckFailedValidation` is enabled.

Reply:
Good call. I decided to just remove "in parallel" since it's an implementation detail the caller doesn't generally need to know about.